Tasks/VsTestPlatformToolInstallerV1/task.loc.json

```diff
@@ -38,7 +38,9 @@
       "groupName": "packageSettings",
       "helpMarkDown": "ms-resource:loc.input.help.packageFeedSelector",
       "options": {
-        "nugetOrg": "Official Nuget"
+        "nugetOrg": "Official Nuget",
```

Comment:
check with prachi -> usually "Official nuget/ Custom feed/Network path"

Reply:
have asked prachi to review strings/experience. She hasn't gotten back yet.